hackclub.com.yaml

```diff
@@ -423,6 +423,17 @@ asiet:
   - ttl: 1
     type: CNAME
     value: kkharidev.github.io.
+asmgame:
+  - ttl: 1
+    type: ALIAS
+    value: hackclub.github.io/asmgame.
```

Comment:
Hey! DNS records can't contain paths (in this case, `/asmgame`). You should make this a CNAME to hackclub.github.io (see docs [here](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain))

Additionally, this TTL is too low. Try a larger value, like 3600 (5 minutes).

Reply:
Isnt 300 5min?

Reply:
fixed these issues